section/ProjectileNewMethods.cpp

```diff
@@ -26,18 +26,13 @@ int GetCurrentTargetPositionXYZ(lua_State *L) {
     if (res.IsFail())
         L->LuaState->Error("%s", res.reason);
     auto projectile = reinterpret_cast<uintptr_t>(res.object);
-    Vector3f pos;
+    Vector3f pos{};
```

Comment:
Is this allocated on the stack, or on the heap?

Reply:
Stack.